specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/publicIpPrefix.json

```diff
@@ -359,6 +359,10 @@
           "format": "int32",
           "description": "The Length of the Public IP Prefix."
         },
+        "loadBalancerFrontendIpConfiguration": {
```

Comment:
Please update examples with this property.
Also, is it backreference or can we specify this value in payload? If the first one, should it be readOnly?

Reply:
`loadBalancerFrontendIpConfiguration` is marked as read-only in backend model, so @khannarheams please add `readOnly: true`